src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,837 @@
+                {{/menu-item-text}}
+              {{/menu-item-main}}
+            {{/menu-item}}
+          {{/menu-list-item}}
+          {{#> menu-list-item}}
+            {{#> menu-item}}
+              {{#> menu-item-main}}
+                {{#> menu-item-text}}
+                  Count
+                {{/menu-item-text}}
+              {{/menu-item-main}}
+            {{/menu-item}}
+          {{/menu-list-item}}
+          {{#> menu-list-item}}
+            {{#> menu-item}}
+              {{#> menu-item-main}}
+                {{#> menu-item-text}}
+                  Labels
+                {{/menu-item-text}}
+              {{/menu-item-main}}
+            {{/menu-item}}
+          {{/menu-list-item}}
+          {{#> menu-list-item}}
+            {{#> menu-item}}
+              {{#> menu-item-main}}
+                {{#> menu-item-text}}
+                  Annotations
+                {{/menu-item-text}}
+              {{/menu-item-main}}
+            {{/menu-item}}
+          {{/menu-list-item}}
+        {{/menu-sublist}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Delete deployment config
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+    {{/menu-list}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-filtering
+```hbs
+{{#> menu}}
+  {{#> menu-header}}
+    {{> form-control controlType="input" input="true" form-control--modifier="pf-m-search" form-control--attribute=(concat 'type="search" id="' id '-search-input" name="' id '-search-input" aria-label="Search"')}}
+  {{/menu-header}}
+  {{> divider}}
+  {{#> menu-body}}
+    {{#> menu-list}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Action 1
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Action 2
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Action 3
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+    {{/menu-list}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-links
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-list}}
+      {{#> menu-list-item}}
+        {{#> menu-item menu-item--type="a" menu-item--attribute='href="#"'}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Link 1
+            {{/menu-item-text}}
+            {{> menu-item-external-icon}}
```

Comment:
I wonder if we should remove the external link icon for at least one of these to show that it isn't necessary unless the icon indicates the link launches in a new window or is an external domain.

Reply:
updated the "links" and "favorites" examples to show only some of the list items displaying the external icon and added `target="_blank"` to menu items with the external icon, does that sound right ?

Reply:
@mcarrano can you confirm what the intent of the "external" icon is?

<img width="251" alt="Screen Shot 2020-08-20 at 10 17 14 AM" src="https://user-images.githubusercontent.com/35148959/90791016-55de6500-e2ce-11ea-9540-355327833ab4.png">